backend/post-service/src/main/java/com/speakapp/postservice/services/ReactionService.java

```diff
@@ -37,4 +41,26 @@ public ReactionType createUpdatePostReaction(ReactionType newReaction, UUID post
         return newReaction;
     }
 
+    public ReactionType createUpdateCommentReaction(ReactionType newReaction, UUID commentId, UUID userId){
+        Comment comment = commentRepository.findById(commentId).orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND,
```

Comment:
może by się dało tutaj zrobić jakąś generic metodę? Chociaż nie wiem na ile to potrzebne tak naprawdę